src/x509.cc

```diff
@@ -45,6 +45,85 @@ std::string parse_args(const Nan::FunctionCallbackInfo<v8::Value>& info) {
   return *String::Utf8Value(info[0]->ToString());
 }
 
+
+
+NAN_METHOD(verify) {
+  Nan::HandleScope scope;
+  OpenSSL_add_all_algorithms();
```

Comment:
@lambrojos May want to make sure that I'm right about `require('crypto')` working the way that I'm thinking it will before you go through the trouble. It may be that we need the preprocessor directive. I'm just hoping that we don't.